router.go

```diff
@@ -187,6 +188,27 @@ func (p *ControllerRegister) addWithMethodParams(pattern string, c ControllerInt
 	route.methods = methods
 	route.routerType = routerTypeBeego
 	route.controllerType = t
+	route.initialize = func() ControllerInterface {
```

Comment:
Looks like there is already bench test for this.
after a little bit testing, having initialize function as nil doesn't affect the results, the only thing that does affect them is removing reflection, which is hard since beego router is basically based on it... =\

Tested on Mac
``` 
goos: darwin
goarch: amd64
```
### Before
```
go test -bench=BenchmarkController -benchmem
BenchmarkController-4            1000000              1408 ns/op             666 B/op          6 allocs/op
```
### After
```
go test -bench=BenchmarkController -benchmem
BenchmarkController-4            1000000              1697 ns/op             682 B/op          8 allocs/op
```